static/css/js-all.less

```diff
@@ -13,6 +13,7 @@
 @import (less) 'components/jquery.autocomplete.less';
 
 @import (less) "components/read-statuses.less";
+@import (less) './components/manage-covers.less';
```

Comment:
is there a reason manage-covers.less is being differently than e.g. `"components/read-statuses.less"`? Can we also use consistent quote-types?